tingbot/error.py

```diff
@@ -43,11 +43,7 @@ def sad_tingbot_image():
 
 
 def error_screen(exc_info):
-    surface = pygame.display.get_surface()
-    if not surface:
-        return
-
-    screen = graphics.Surface(surface)
+    screen = graphics.screen
 
```

Comment:
If you generate an error message in the simulator then (under linux at least) then the error message is printed in the wrong place with the previous setup. This probably doesn't happen on the mac as pygame.display.get_surface only gives you the screen area, whereas on windows and linux it gives you the whole simulator area.